src/components/picker/picker.tsx

```diff
@@ -221,10 +221,6 @@ export class Picker {
 
     public componentWillLoad() {
         this.chips = this.createChips(this.value);
-        /* eslint-disable-next-line no-console */
-        console.warn(
-            "The `iconColor` prop is deprecated now! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
-        );
```

Comment:
isn't it enough that we have the `console.warn` in limel-chipset? Since picker is using Chipset anyways?